src/test/resources/hydration.json

```diff
@@ -61,7 +60,8 @@
                         "sourceType": "FIELD_ARGUMENT"
                       }
                     }
-                  ]
+                  ],
+                  "objectIdentifier": "id"
```

Comment:
why did we needed to change/add that?

Reply:
I moved it down there in the previous grammar, but it yeah you're right its fine to go back